distributedsocialnetwork/post/retrieval.py

```diff
@@ -177,6 +177,61 @@ def get_detailed_post(post_id):
     # response = requests.get(
     #             url, headers={'content-type': 'application/json', 'Accept': 'application/json'})
 
+def get_comments(pk):
+    #return Comment.objects.filter(post_id=post_id)
+    comments = Comment.objects.filter(post_id=pk)
+    comm_ids = []
+    for comment in comments:
+        comm_ids.append(comment["id"])
+    nodes = list(Node.objects.all())
```

Comment:
This doesn't make any sense, why are we querying every node if we know which node this post belongs to? We only need to talk to one.